react-app/src/face-rec.js

```diff
@@ -151,6 +151,7 @@ function removeFileExt (fileName) {
 async function loadReferenceImages (paths) {
     const descriptors = []
 
+    // For reference images already in public folder
```

Comment:
You can just remove this and the `public/images` folder in general. We only needed it for that basic testing. Removing it should actually speed up the facial recognition quite a bit too.